python/MaterialXTest/genshader.py

```diff
@@ -49,7 +49,7 @@ def test_ShaderInterface(self):
         nodeDef = doc.addNodeDef("ND_foo", "color3", "foo")
         fooInputA = nodeDef.addInput("a", "color3")
         fooInputB = nodeDef.addInput("b", "color3")
-        fooOutput = nodeDef.addOutput("o", "color3")
+        fooOutput = nodeDef.getOutput("out")
```

Comment:
Why is the type removed?

Reply:
Type removed?

Reply:
Sorry, why is addOutput() with type removed? This should be allowed to be tested.

Reply:
It's a different method, it's getOutput, not addOutput.

Reply:
Think you missed the point. addOutput can be called and checked for failure now.